tests/validation/tests/v3_api/scripts/pipeline/Jenkinsfile_ontag_matrix

```diff
@@ -172,7 +185,7 @@ if ( true == via_webhook() && (!(rancher_version ==~ rancher_version_regex)) ) {
                       }
                       else if (rancher_version.startsWith("v2.7")){
                         if (!env.RKE_VERSION) {
-                          RKE_VERSION = "v1.4.0-rc4"
+                          RKE_VERSION = "v1.4.0"
                         }
                         if (!env.RANCHER_K3S_VERSION) {
                           RANCHER_K3S_VERSION = "v1.24.6+k3s1"
```

Comment:
could you update this to the latest patch while you're here (or 1.25 if PSA won't mess anything up)?